src/store.js

```diff
@@ -10,7 +10,8 @@ const persistConfig = {
     storage: storage,
     // autoMergeLevel1 is the default anyway, but I want to be explicit
     stateReconciler: autoMergeLevel1,
-    whitelist: ['auth', 'map', 'addEditSite', 'network']
+    whitelist: ['auth', 'map', 'addEditSite', 'network'],
+    debounce: 1000
```

Comment:
This can probably be removed. In my debugging to try to figure out why the app was crashing, I added a debounce to the persistor of 1s. I'll probably take this out in the next build unless it seems like it's actually helping.